contracts/ColonyFunding.sol

```diff
@@ -60,11 +60,26 @@ contract ColonyFunding is ColonyStorage, DSMath {
     return task.payouts[_role][_token];
   }
 
+  function moveFailedTaskFunds(uint256 _id, address _token) public
```

Comment:
Instead of introducing a new function specifically for this, can we simply set the Worker payout to `0` in `finalizeTask`? Then we can use `moveFundsBetweenPots` to refund anything left after everyone's claimed?

Reply:
Ok, setting the worker rating to 0 after reputation update